### PR TITLE
[[ Bug 15315 ]] Insert the appended chunk delimiters at the end of the f...

### PR DESCRIPTION
**Pull request by livecodesebastien**
...ield instead of resetting its whole text

### CODE REVIEW

File: engine/src/exec-interface.cpp [hunk 1]
```diff
@@ -3234,14 +3234,16 @@ void MCInterfaceExecPutIntoField(MCExecContext& ctxt, MCStringRef p_string, int
...
+        t_start_index -= p_chunk.mark.changed;
+        if (!MCStringCopySubstring((MCStringRef)p_chunk . mark . text, MCRangeMake(t_start_index, p_chunk.mark.changed), &t_string))
             return;
         
-        // in this case the chunk indices will be correct whatever the preposition
-        /* UNCHECKED */ MCStringReplace(*t_string, MCRangeMake(p_chunk . mark . start, p_chunk . mark . finish - p_chunk . mark . start), p_string);
-        
-        p_chunk . object -> setstringprop(ctxt, p_chunk . part_id, P_TEXT, False, *t_string);
-        return;
+        t_field -> settextindex(p_chunk .part_id, t_start_index, t_start_index, *t_string, False);
```
Comment (runrevmark) [inline engine/src/exec-interface.cpp]: Do you mean to fall-through to the remainder of the outer clause?

**Comment by livecodesebastien** [comment]
Pull request updated

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 3f97eaf

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 3f97eaf6717cf45e937524d9fd7d7f0e1a753bcc

**Pull request merged**